protocol/src/main/contraband-scala/sbt/internal/bsp/ResourcesItem.scala

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
I wonder why those generated files are commited, any idea ?

Reply:
It is probably a safety net to avoid breaking compatibility between versions